app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -85,46 +69,56 @@ To be eligible for a scholarship, you’ll need at least one of the following:
 * a master’s degree
 * a PhD
 
-In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
+If you achieved a 2:2 in your degree you may get a scholarship if you have enough relevant experience.
+
+Scholarship providers set their own:
 
-#### Benefits
+* eligibility requirements 
+* application deadlines
 
-Scholarships are provided by charitable organisations with a passion for their subject area.
+#### How they're paid
 
-They offer more than just financial benefits, including:
+A scholarship is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-* subject specialist events and webinars
-* online and face-to-face workshops
-* mentoring and coaching
-* a community of ex-scholars, teachers and subject leaders
-* access to classroom resources
-* networking events during your teacher training and beyond
+You will not need to pay tax or National Insurance on scholarship payments. 
 
 #### How to apply
 
-Scholarships are offered by independent institutions. They set their own eligibility criteria and you’ll need to apply through the relevant scholarship body:
+You need to apply directly to the scholarship provider via their website. They will confirm if you meet their eligibility requirements. 
+
+If you get a scholarship, you need to tell your teacher training provider.   
 
 * [the Royal Society of Chemistry](https://www.rsc.org/prizes-funding/funding/find-funding/teacher-training-scholarships/) (chemistry)
 * [BCS The Chartered Institute for IT](https://www.bcs.org/get-qualified/certification-and-scholarships-for-teachers/bcs-computer-teacher-scholarships/) (computing)
 * [British Council](https://www.britishcouncil.org/education/he-science/opportunities/ltts) (languages -- French, German and Spanish only)
 * [the Institute of Mathematics and its Applications, and partners](https://teachingmathsscholars.org/home) (maths)
 * [the Institute of Physics](https://www.iop.org/about/support-grants/iop-teacher-training-scholarships#gref) (physics)
 
-Each scholarship body will have its own deadline for applications. You can find these on their individual websites.
+## Compare bursaries and scholarships
```

Comment:
I'm not sure `Compare` is the right word here - there are far fewer scholarships on offer?